library/src/main/kotlin/org/codice/compliance/SAMLComplianceException.kt

```diff
@@ -62,6 +66,18 @@ class SAMLComplianceException : Exception {
             }
         }
 
+        fun recreateExceptionWithErrorMessage(exception: SAMLComplianceException):
+            SAMLComplianceException {
+            return if (exception.cause != null) {
+                SAMLComplianceException(
+                    "${exception.message}$IDP_ERROR_RESPONSE_REMINDER_MESSAGE",
+                    exception.cause)
+            } else {
+                SAMLComplianceException(
+                    "${exception.message}$IDP_ERROR_RESPONSE_REMINDER_MESSAGE")
```

Comment:
❓ Would we like a space or a carriage return or some other break between these two strings?